### DISCUSSION TITLE
Regions for all friends

### DISCUSSION THREAD

**Discussion by Dab942** [owntracks/android#1672] [Q&A]
When adding a region, is that done through the imported file, each time, or is there a way to add those places in the app?   Home and Work only show for me and notifications are only seen by my device.    From the booklet, it seems like that info needs to be in the individual files to be imported to each user's device?

**Comment by jpmens** [comment]
It is correct that regions need to be define on each device which should report events for a particular regions. There's no "central database" of regions.

**Reply by Dab942** [reply]
Is there a way for all "friends" to get the notifications when entering/leaving common "regions"?

**Comment by jpmens** [comment]
Not without republishing, say ,over MQTT, unless all friends have the same region configured.

**Reply by Dab942** [reply]
That's what I'm looking to do.   I have MQTT implementation.   I have other family members that I'd like to track - similar to life360 -- I also want to link to Hubitat for automation.

**Comment by jpmens** [comment]
closing as answered
